### DISCUSSION TITLE
How to move the actual files from one Nhost project to another?

### DISCUSSION THREAD

**Discussion by vipafattal** [nhost/nhost#2542] [Q&A] [answered]
I used pg_dump & pg_restore to copy db content from one nhost project to another, however that doesn't copy the actual files, any help on that? Thanks

**Comment by dbarrosop** [comment]
We don't have a tool for this but should be fairly straightforward to write one. Basically:

1. List files from the source project using a graphql query 
2. Iterate over the files:
   1. Check if the file is present already in the destination
   2. Download from source
   3. Upload to source specifying the [id](https://github.com/nhost/hasura-storage/blob/main/example_curl.sh#L23)

If you want to contribute such tool don't hesitate to let us know.

**Reply by vipafattal** [reply, marked as answer]
@dbarrosop I have created a command-line tool to copy storage from one nhost project to another if you're interested it's here:
https://github.com/vipafattal/nhost_storage_copify/
